html5/connect.html

```diff
@@ -1618,7 +1613,7 @@ <h4 class="panel-title">Advanced options</h4>
         });
         //delete session params:
         try {
-          sessionStorage.clear();
+          sessionStorage.removeItem(Utilities.getSessionStoragePrefix());
```

Comment:
All the other calls now go through `Utilities` to access `sessionStorage` so this one should do too. Please add a `clearSessionStorage` function there.